src/main/webapp/index.html

```diff
@@ -1,46 +1,34 @@
 <!DOCTYPE html>
 <html>
-<head>
-    <meta charset="utf-8"/>
+  <head>
+    <meta charset="utf-8" />
```

Comment:
The style suggests not closing void elements (here and below)

https://google.github.io/styleguide/htmlcssguide.html#HTML_Validity